cli/src/cli_util.rs

```diff
@@ -573,9 +591,7 @@ impl AdvanceBranchesSettings {
 /// Provides utilities for writing a command that works on a [`Workspace`]
 /// (which most commands do).
 pub struct WorkspaceCommandHelper {
-    string_args: Vec<String>,
-    global_args: GlobalArgs,
-    settings: UserSettings,
+    command: CommandHelper,
```

Comment:
nit: `command.data` could be cloned instead, but I have no idea which is better overall.

Reply:
I'm not sure either. By keeping the whole `CommandHelper` here, we can at least use its public interface, which I have started doing.